docs/Fdroid.md

```diff
@@ -0,0 +1,23 @@
+# Trustroots App on F-Droid
+
+F-Droid users can manually add additional repositories in order to receive applications. Below repository provides the Trustroots app to F-Droid users who wish to avoid the Play Store version.
+
+## Issue
+
+We don't have resources yet to build a separate "native app" for Android so we're using Expo.io framework instead.
+
+Using Expo seems to block inclusion to the official F-Droid repository. As an alternative, a private repository has been set up to serve F-Droid users with the latest Trustroots Android mobile app.
+
+## Repository details
+
+```html
```

Comment:
Maybe just ``` without HTML since there's no HTML to syntax highlight?

Reply:
Did in https://github.com/Trustroots/trustroots/pull/1046/commits/ea68a5eb728ceb55d99f466c821244f33d3b7ad5